data-access/persoon.ts

```diff
@@ -102,6 +103,44 @@ export async function checkPersonExistsAllGraphs(
   return getBooleanSparqlResult(result);
 }
 
+export async function createrPersonFromUri(
+  personUri: Term,
+  firstname: Term,
+  lastname: Term,
+  graph: Term,
+): Promise<void> {
+  const uuidOfPersoon = findUuidFromUri(personUri.value);
```

Comment:
I'm not really a big fan of this. But I get why you do it like this. Problem is your uuid does not have to be included in the uri. This is something we try to do, but it is not a strict requirement. 
Also what if the uuid does not match your specific regex, how sure are we that a uuid will have that form? I think it might be better to just get whatever comes after the prefix `http://data.lblod.info/id/personen/` . I feel like this should be a better match. 
And I think if you can not find the uuid, you should just generate a new one. Since all relations to this person are based on the uri, this uuid is a property we can set to whatever we like.

Reply:
I tested it with these two URI's: 'http://data.lblod.info/id/personen/6149989FE575460008000032', 'http://data.lblod.info/id/personen/3bae297a-5544-4218-a142-97061a2cf0bb'

Should i maybve leave the check and add a method in the util to get the ID form the URI but that we first check on UUID and if not exist everything after the last slash character. What do you think?